stable/kube-lego/values.yaml

```diff
@@ -4,7 +4,7 @@
 config:
   ## Email address to use for registration with Let's Encrypt
   ##
-  # LEGO_EMAIL: my@email.tld
+  LEGO_EMAIL: my@email.tld
```

Comment:
This should remain commented, as people weren't paying attention to the fact that the email address had to be changed. Keeping it unset by default means deployment fails unless the user provides a value.

Ref: https://github.com/kubernetes/charts/pull/912

Reply:
I just edited this because otherwise I get a linting error. Will it pass the CI tests if I revert it?

Reply:
Unfortunately the tests won't pass, but we know about & expect the failure.